monday/query_joins.py

```diff
@@ -167,6 +166,75 @@ def delete_item_query(item_id):
     return query
 
 
+# COLUMNS RESOURCE QUERIES
+def create_column(
+    board_id: int, column_title: str, column_type: None, defaults: dict[str, any] = None, description: str = None
+):
+    """C```reate a new column by board ID
+
+    Args:
+        board_id [int]: Board ID
+        column_title [str]: Column's title
+        column_type [ColumnTypes]: Column's type
+        defaults [Dict[str,Any]]: Default value
+        description [str]: Column's description
+
+    Return:
+        [Dict[str,Any]]: Column's Monday details after creation
+    """
+    defaults = defaults if defaults else ""
+    description = description if description else ""
+
+    if not column_type:
+        query = """mutation{
+            create_column(board_id: %s, title: "%s") {
+                id
+                title
+                description
+            }
+        }""" % (
+            board_id,
+            column_title,
+        )
+    else:
+
+        query = """mutation{
+            create_column(board_id: %s, title: "%s", description: "%s", column_type: %s, defaults: %s) {
```

Comment:
Just a warning as the column_type seems to have been meant to be typed to `ColumnTypes`, string interpolation of enums has changed somewhere in 3.10 I believe:

```python
class A(Enum):
  ABC = "123"
"""enum value: %s""" % (A.ABC,) # 'From enum value: A.ABC'
"""enum value: %s""" % (A.ABC.value) # correct: enum value: 123

```